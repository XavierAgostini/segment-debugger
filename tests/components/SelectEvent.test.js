import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SelectEvent from '../../src/components/SelectEvent'

describe('<SelectEvent/>', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<SelectEvent/>)
      const component = wrapper.dive()
      expect(toJson(component)).toMatchSnapshot()
    })
  })
})