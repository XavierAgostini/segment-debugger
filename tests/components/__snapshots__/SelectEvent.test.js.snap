// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectEvent/> should render SelectEvent as expected 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
